include/Partitioning/CoordinatedRecord.hpp

```diff
@@ -125,7 +125,7 @@ bool CoordinatedRecord<T>::releaseLock() {
 }
 template <typename T>
 int CoordinatedRecord<T>::getReplicas() const {
-  return partitions.size();
+  return (int)partitions.size();
```

Comment:
This explicit cast I think is useless, we can return a `size_t` type ( as soon as possibile we will release a major)